src/database/Connector.java

```diff
@@ -0,0 +1,192 @@
+
+  /**
+   * print all the current useNnames
+   *
+   */
+  public List<String> getAllUsers() {
+
+    String sqlSelectUsername = "select username from " + tableName;
+    LinkedList<String> userNames = new LinkedList<>();
+    try (Connection conn = ds.getConnection()) {
+      try (PreparedStatement getUsers = conn.prepareStatement(sqlSelectUsername)) {
+        try (ResultSet users = getUsers.executeQuery()) {
+          while (users.next()) {
+            userNames.add(users.getString("username"));
+          }
+          return userNames;
+        }
+      }
+    } catch (SQLException e) {
+      e.printStackTrace();
+      return  null;
+    }
+
+  }
+
+  /**
+   * AddAccount is to add the new account to the database.
+   * 
+   * @param username
+   * @param password
+   * @return true if the insertion is successful and complete; false, if the insertion fails
+   */
+  public boolean addAccount(String username, String password) {
+
+    String sqlInsertAccount = "insert into "+tableName + "(username, password)" + "values(?,?)";
+    try (Connection conn = ds.getConnection()) {
+      try(PreparedStatement insertAccount = conn.prepareStatement(sqlInsertAccount)) {
+        insertAccount.setString(1, username);
+        insertAccount.setString(2, password);
+        insertAccount.executeUpdate();
+        return true;
+      }
+    }
+    catch (SQLException e) {
+      e.printStackTrace();
+      return false;
+    }
+  }
+
+  /**
+   * clean all the data inside the database
+   *
+   * @return true if the data has been cleaned
+   */
+  public boolean dropAllAccounts() {
+
+    String sqlDrop = "truncate table " + tableName;
+    try (Connection conn = ds.getConnection()) {
+      try (PreparedStatement dropAll = conn.prepareStatement(sqlDrop)) {
+        dropAll.executeUpdate();
+        //System.out.println("the table has been cleared");
+        return true;
+      }
+    }
+    catch (SQLException e) {
+      e.printStackTrace();
+      return false;
+    }
+  }
+
+  /**
+   * verify if the account username and password input by users match what has been recorded in
+   * database
+   * 
+   * @param username
+   * @param password
+   * @return true if the account is verified, else, false, if the account is not valid
+   */
+  public boolean verifyAccount(String username, String password) {
+
+    String sqlSelectPassword = "select password from " + tableName + " where username = ?";
+    try (Connection conn = ds.getConnection()) {
+      try (PreparedStatement selectPassword = conn.prepareStatement(sqlSelectPassword)) {
+        // the account exists, check password
+        selectPassword.setString(1, username);
+        try (ResultSet resultPassword = selectPassword.executeQuery()) {
+          if (resultPassword.next()) {
+            // get the stored password from database
+            String passwordInDB = resultPassword.getString(1);
+            // password does not match
+            if (!passwordInDB.equals(password)) {
```

Comment:
Are you guys storing the password in plaintext? If so this is a feature to improve :)